README.md

```diff
@@ -1 +1,75 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Project Name
+
+    # Portfolio-setup-and-mobile-version-skeleton
+
+> Description the project.
+
+    This project is designed to help the develop gain mastery of
+    1.   Figma design to create a UI.
+    2.   understanding use of Flexbox to place elements in the page.
+    3.   Build a personal portfolio site.
+    4.   Use images and backgrounds to enhance the look of the website
+
+## Built With
+
+- Major languages
+  CSS
+  HTML
+- Frameworks
+- Technologies used
+  Git and GitHub
+  Visual Studio Code
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how developerst can work together remotely on a project using technology like Git and GitHub .**
+**Project are collect are gotten gotten fron a centre source, worked upon and megred with the origin project after approval**
+**Developers portfolio is also built using Figma, html and css technologies**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+    knowledge on Git, Html and Css
+
+### Setup
+
+    Repository on your Machine
+
+### Install
+
+    Visual Studio Code
+
+### Deployment
+
+## Authors
```

Comment:
- [ ] [OPTIONAL] Kindly take out the parts of the README file that are not been used in the project in order for you to have a clean code base.